watson/cli.py

```diff
@@ -939,3 +939,65 @@ def merge(watson, frames_with_conflict, force):
     watson.frames = original_frames
     watson.frames.changed = True
     watson.save()
+
+
+@cli.command()
+@click.option('-p', '--project', default=None, nargs=2, help="Rename project")
+@click.option('-T', '--tag', default=None, nargs=2, help="Rename tag")
+@click.pass_obj
+def rename(watson, project, tag):
+    """
+    Renames a project or tag.
```

Comment:
Help message should be written in imperative form, i.e. "Rename tag..." instead of "Renames"". And please be consistent with the order of "project or tag" here and in the zsh completion file.